composer.json

```diff
@@ -11,7 +11,7 @@
         "spryker/symfony": "^3.5.0"
     },
     "require-dev": {
-        "codeception/module-asserts": "^1.3.0",
+        "codeception/module-asserts": "^1.3.0 || ^3.0.0",
```

Comment:
Why do we need to support both versions?

Reply:
for compatibility Codeception version 4 and 5
https://github.com/Codeception/module-asserts/blob/3.0.0/composer.json#L30

Reply:
We keep codeception 5 support only, please drop 1.3.0 constraint.